src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedXml.cs

```diff
@@ -232,6 +241,24 @@ private byte[] GetCipherValue(CipherData cipherData)
             throw new CryptographicException(SR.Cryptography_Xml_MissingCipherData);
         }
 
+        private void DownloadCipherValue(CipherData cipherData, out Stream inputStream, out Stream decInputStream, out WebResponse response)
+        {
+            // maybe a network stream, make sure we allow just what is needed!!
+            PermissionSet ps = SecurityManager.GetStandardSandbox(_evidence);
+            ps.PermitOnly();
```

Comment:
@danmosemsft what's the strategy for code that checks if permission is granted? I guess it's intentional that PermitOnly wasn't stubbed out silently instead of throwing? Maybe conditional compilation?